app/jobs/subscription_confirm_job.rb

```diff
@@ -57,10 +57,13 @@ def process_payment!(order)
     return true unless order.payment_required?
 
     setup_payment!(order)
-    return false if order.errors.present?
+    return false if order.errors.any?
```

Comment:
Why do you prefer `any?` over `present?` here?

Reply:
I change it here just for consistency with the calls in the same method.